xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -248,11 +248,11 @@ constexpr int DEFAULT_LATENCY_DIVIDER = 3;
 
 using namespace AE::SINK;
 
-std::unique_ptr<PIPEWIRE::CPipewire> pipewire;
+std::unique_ptr<KODI::PIPEWIRE::CPipewire> pipewire;
```

Comment:
It's perfectly fine to add `using namespace KODI;` at the top of any Kodi cpp file, to simplify these lines and not require a change if AE is ever moved into namespace `KODI`.

Reply:
updated, thanks!